apps/application/serializers/chat_message_serializers.py

```diff
@@ -125,7 +126,9 @@ def append_chat_record(self, chat_record: ChatRecord, client_id=None):
             # 插入数据库
             if not QuerySet(Chat).filter(id=self.chat_id).exists():
                 Chat(id=self.chat_id, application_id=self.application.id, abstract=chat_record.problem_text[0:1024],
-                     client_id=client_id).save()
+                     client_id=client_id, update_time=datetime.now()).save()
+            else:
+                Chat.objects.filter(id=self.chat_id).update(update_time=datetime.now())
             # 插入会话记录
             chat_record.save()
 
```

Comment:
在上面给出的代码片段中，我发现了以下问题和改进建议：

问题：
1. 缺乏函数名规范化：从 "append_chat_record" 搬迁到 "add_chat_recorder"
2. 在处理 UUID 对象时存在错误。Python 中，使用 str(uuid.uuid()) 获取的是字符串形式的 UUID。
3.

Please provide more details about the specific issues you're referring to, but I think it might be beneficial for your project to refactor `add_chat_recorders` method into several smaller functions that can be tested individually or combined efficiently with other operations depending on their dependencies and constraints.

For example:

- A function named `convert_uuid`.
- A function named `insert_chat_records`.

By doing so,
- You avoid having multiple lines of identical code that are difficult to read and maintain.
- Your implementation becomes cleaner and easier to test independently.
- Improvements like using Django's built-in models (specifically 'Chat'), making string comparisons, and handling potential edge cases (like empty list insertion) would also likely enhance overall quality and reliability of code.